classes/fields/pick.php

```diff
@@ -168,6 +168,28 @@ public function options() {
 				),
 				'dependency' => true,
 			),
+			static::$type . '_display_format_multi'   => array(
```

Comment:
:warning: Array double arrow not aligned correctly; expected 5 space(s) between "'_display_format_multi'" and double arrow, but found 3.

<sub>hash: eb129162016f1041c7c34d2d0bc054d7</sub>